common/kommander-operator/helmrelease.yaml

```
@@ -12,7 +12,7 @@ spec:
         kind: GitRepository
         name: management
         namespace: kommander-flux
-      version: 2.7.0-dev
+      version: 0.2.0
```

should this be bumped?
```suggestion
      version: 0.3.0
```

---

oops yes thank you!

---

fixed in https://github.com/mesosphere/kommander-applications/pull/1568/commits/36f693de7d25ec2852de88ffa3d84db712cc74bc